tr/messages.json

```diff
@@ -1 +1,430 @@
-{}
+{
+	"extensionDescription": {
+		"message": "Cümleler arasındaki uzun duraksamaları ileri sararak ~ 1,5 kat daha hızlı izleyin."
+	},
+	"enable": {
+		"message": "Aktif"
+	},
+	"more": {
+		"message": "Daha fazla"
+	},
+	"volume": {
+		"message": "Ses\nBu, malzemenin mevcut ses yüksekliği değil, TV'deki gibi amplifikasyon çarpanıdır."
+	},
+	"timeSaved": {
+		"message": "Zaman tasarufu bilgisi"
+	},
+	"overTheLast": {
+		"message": "Son $duration$ içinde",
+		"placeholders": {
+			"duration": {
+				"content": "$1",
+				"example": "15:00"
+			}
+		}
+	},
+	"numbersMeanings": {
+		"message": "Sayıların anlamları (sırasıyla):"
+	},
+	"timeSavedComparedToSounded": {
+		"message": "Videonun ses hızına kıyasla ortalama olarak ne kadar hızlı oynatıldığı"
+	},
+	"timeSavedComparedToIntrinsic": {
+		"message": "Ortalama olarak, videonun normal (gerçek) hıza kıyasla etkin bir şekilde ne kadar hızlı oynatıldığı"
+	},
+	"timeSavedComparedToSoundedAbs": {
+		"message": "Videoyu sadece ses hızında oynatmaya kıyasla kazanılan zaman"
+	},
+	"timeSavedComparedToIntrinsicAbs": {
+		"message": "Videoyu normal (gerçek) hızda oynatmaya kıyasla kazanılan zaman"
+	},
+	"wouldHaveLastedIfSpeedWasSounded": {
+		"message": "Sessizliği ATLAMADAN ses hızında oynatma ne kadar sürer"
+	},
+	"wouldHaveLastedIfSpeedWasIntrinsic": {
+		"message": "Sessizliği ATLAMADAN normal (gerçek) hızda oynatma ne kadar sürer"
+	},
+	"timeSavedPercentage": {
+		"message": "Eşdeğer zaman tasarrufu yüzdesi:"
+	},
+	"comparedToSounded": {
+		"message": "Ses ile karşılaştırıldığında"
+	},
+	"comparedToIntrinsic": {
+		"message": "Normale kıyasla (gerçek)"
+	},
+	"noSuitableElement": {
+		"message": "Sayfada uygun bir medya öğesi bulunamadı"
+	},
+	"retry": {
+		"message": "Tekrar"
+	},
+	"alsoSearchFor": {
+		"message": "Ayrıca $type$ öğelerini arayın",
+		"placeholders": {
+			"type": {
+				"content": "$1",
+				"example": "ses"
+			}
+		}
+	},
+	"changeElementSearchCriteria": {
+		"message": "Arama kriterlerini değiştir…"
+	},
+	"audio": {
+		"message": "ses"
+	},
+	"video": {
+		"message": "video"
+	},
+	"contentScriptFail": {
+		"message": "Bu sayfadaki içerik komut dosyası yürütülemedi"
+	},
+	"suggestOpenLocalFile": {
+		"message": "**$openLocalFile$**?mak mı istiyorsunuz?",
```

Comment:
Does this look right? 
![image](https://user-images.githubusercontent.com/39462442/173080126-63a8202f-fc77-4757-aa84-5ea20d5aa4d6.png)